ext/reflection/php_reflection.c

```diff
@@ -6571,7 +6803,6 @@ PHP_MINIT_FUNCTION(reflection) /* {{{ */
 	/* IS_IMPLICIT_ABSTRACT is not longer used */
 	REGISTER_REFLECTION_CLASS_CONST_LONG(class, "IS_IMPLICIT_ABSTRACT", ZEND_ACC_IMPLICIT_ABSTRACT_CLASS);
 	REGISTER_REFLECTION_CLASS_CONST_LONG(class, "IS_EXPLICIT_ABSTRACT", ZEND_ACC_EXPLICIT_ABSTRACT_CLASS);
-	REGISTER_REFLECTION_CLASS_CONST_LONG(class, "IS_FINAL", ZEND_ACC_FINAL);
```

Comment:
Similarly, why was ReflectionClass::IS_FINAL removed?

Reply:
Oops, that was a mistake. I'll re-add it.